define/types.go

```diff
@@ -260,13 +260,6 @@ func parseGitBuildContext(url string) (string, string, string) {
 	return gitBranchPart[0], gitSubdir, gitBranch
 }
 
-func isGitTag(remote, ref string) bool {
```

Comment:
There is no need to for this because `fetch` below automatically fails if `reference` does not exists.